### DISCUSSION TITLE
AG Plugin does not always let me open the UI of my VST instruments. Remote UI stops working.

### DISCUSSION THREAD

**Discussion by wpostma** [apohl79/audiogridder#1183] [Help]
I'm using AG Plugin within my Bitwig daw on a mac (on M1) talking to an AG Server on Windows 10.
The first plugin I tried is IK Multimedia SampleTank since having that offloaded to a remote machine would save me a lot of RAM and CPU.

The UI is rather sluggish and then stops working at all.  Once it's stopped showing me the UI I have to restart the AG Server to get UI to work again.

Is there logging I can enable on the server side that might help figure out where the Remote UI feature is going wrong?

Update: Going to follow the bug-reports link below and add logging and details.

**Comment by apohl79** [comment]
If you could follow these steps, thats would be great!

https://audiogridder.com/bug-reports/

Cheers!
